Sources/SymbolKit/SymbolGraph/Symbol/Symbol.swift

```diff
@@ -331,12 +333,52 @@ extension SymbolGraph.Symbol {
                 case .typealias      : return "typealias"
                 case .var            : return "var"
                 case .module         : return "module"
-                case .unknown        : return "unknown"
+                case .unknown(let k) : return k
                 }
             }
         }
 
-        // FIXME: Save "unknown" symbol kinds in a synchronized set to prevent loss of data (rdar://84276085)
+        private static let StarterCases: [Self] = [
```

Comment:
Can we please document this? Not immediately obvious what this refers to

```suggestion
        private static let starterCases: [Self] = [
```